lib/gobstones/batch_parser.rb

```diff
@@ -3,14 +3,18 @@ class << self
     def parse(request)
       test = parse_test(request)
 
-      options = parse_options test
+      options = parse_options(test).merge(parse_settings(request))
       examples = parse_examples test, options
 
       Gobstones::Batch.new request.content, examples, request.extra, options
     end
 
     private
 
+    def parse_settings(request)
+      { game_framework: request.settings.try(:fetch, 'game_framework') || false }
```

Comment:
I think using `&` + `dig` may be clearer and more robust since it does not throw an error if key is absent: `!!request.settings&.dig(:game_framework)`

Reply:
:mage_man:

Reply:
Is `&.` a null-safe method call?

Reply:
In fact, we can go a step further using `dig`:

```ruby
!!request.dig(:settings, 'game_framework')
```

(mixing symbols and strings doesn't look so fine... but symbolizing the hash doesn't either seems very good).